extensions/sql-database-projects/src/models/project.ts

```diff
@@ -499,20 +499,30 @@ export class Project implements ISqlProject {
 		const result = await this.sqlProjService.addFolder(this.projectFilePath, relativeFolderPath);
 		this.throwIfFailed(result);
 
+		// Note: adding a folder does not necessarily mean adding the contents of the folder.
+		// SDK projects must still adjust their globs, and Legacy projects must still include each file.
```

Comment:
when would an SDK project need to adjust their globs? If the folder added is outside the project's folder? Or if they have an exclude glob?

Reply:
Yeah, either of those (I know external folders aren't included right now).  I just wanted to say that presence of `<Folder Include>` entries doesn't change the files that get included; I can reword the comment to be more clear.